modules/sdk-coin-atom/src/lib/utils.ts

```diff
@@ -39,9 +39,18 @@ export class Utils implements BaseUtils {
 
   /** @inheritdoc */
   isValidTransactionId(txId: string): boolean {
-    throw new Error('Method not implemented.');
+    return this.validateBlake2b(txId);
   }
 
+  validateBlake2b(hash: string): boolean {
+    if (!hash) {
+      return false;
+    }
+    if (hash.length !== 64) {
+      return false;
+    }
```

Comment:
they can be merged into `if(!hash || hash.length !== 64)` or `if(hash?.length !== 64)`

Reply:
updated !!